scripts/netContain/release_image.sh

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 
+set -euo pipefail
+
 contiv_version=""
 docker_user="contiv"
```

Comment:
'contiv' user should not be used as default, as it is not valid anymore. If it is not used, please remove